src/group_theory/subgroup.lean

```diff
@@ -779,6 +779,11 @@ lemma mem_map {f : G →* N} {K : subgroup G} {y : N} :
   y ∈ K.map f ↔ ∃ x ∈ K, f x = y :=
 mem_image_iff_bex
 
+@[to_additive]
+lemma mem_map_of_mem {K : subgroup G} (f : G →* N) {x : G} (hx : x ∈ K) :
+  f x ∈ subgroup.map f K :=
+subgroup.mem_map.mpr ⟨x, hx, rfl⟩
```

Comment:
mem_image_of_mem
```suggestion
mem_image_of_mem f hx
```

Reply:
More concerningly, this lemma statement does not match [=`submonoid.mem_map_of_mem`](https://leanprover-community.github.io/mathlib_docs/group_theory/submonoid/operations.html#submonoid.mem_map_of_mem), which is stated with `(x : K)` instead of `{x : G} (hx : x ∈ K)`.